doc/install/containers.rst

```diff
@@ -0,0 +1,86 @@
+Ceph Container Images
+=====================
+
+Official Releases
+-----------------
+
+Ceph Container images are available from Docker Hub at::
+
+  https://hub.docker.com/r/ceph
+
+
+ceph/ceph
+^^^^^^^^^
+
+- General purpose Ceph container with all necessary daemons and
+  dependencies installed.
+
++------------------+--------------------------------------------------------------+
+| Tag              | Meaning                                                      |
++------------------+--------------------------------------------------------------+
+| vRELNUM          | Latest release in this series (e.g., *v14* = Nautilus)       |
++------------------+--------------------------------------------------------------+
+| vRELNUM.2        | Latest *stable* release in this stable series (e.g., *v14.2*)|
++------------------+--------------------------------------------------------------+
+| vRELNUM.2.DOTNUM | A specific release (e.g., *v14.2.4*)                         |
```

Comment:
This doesn't exist yet

Reply:
can we add it? should i create an issue?